website/source/docs/concepts/tokens.html.md

```diff
@@ -42,14 +42,61 @@ Finally, the `-policy` flag can be used to set the policies associated
 with the token. Learn more about policies on the
 [policies concepts](/docs/concepts/policies.html) page.
 
+<a name="ttls-and-leases" />
 ## Token Time-To-Live and Leases
 
-Every non-root token has a time-to-live (TTL) associated with it.  After the
-TTL is up, the token will no longer function. In addition to no longer
-functioning, Vault will revoke it.
+Every non-root token has a time-to-live (TTL) associated with it, which is
+the number of seconds the token is valid for from this moment in time.
+The TTL is constantly decreasing, and Vault will revoke the token when the TTL hits 0.
+You can find out how much of a token's TTL remains using the `vault token-lookup` command.
 
-When a token is revoked, any leases associated with the token will be revoked
-as well, even if the TTLs on the individual leases are not yet up. For example,
-if a user requests AWS access keys, after the token expires the AWS access keys
-will also be revoked. In order to avoid your token being revoked, the `vault
-token-renew` command should be periodically used to renew the token.
+When a token is revoked, any leases associated with the token will also be revoked,
+even if the TTLs on the individual leases are not yet up. For example,
+if a developer uses a token to generate AWS access keys, the AWS access keys
+will be revoked when the token is revoked. The developer will then need to
+create a new token, and use it to request a new set of AWS access keys.
+
+To stop your token being revoked, you should call the `vault
+token-renew` command periodically, which will attempt to extend the token's TTL.
+You can specify _the new TTL_ using the `increment` argument. If this is
+not specified, then Vault will use the default TTL specified by the
+`auth/token` mount, or the global [`default_lease_ttl`](/docs/config/index.html#default_lease_ttl).
+
+**NOTE:** Vault enforces an upper bound on how long non-root tokens can exist.
+This bound is called the "max lease TTL", and is measured in hours from
+the moment the token was created. Vault will not allow a token to be renewed
+beyond this point. If you attempt to do so, Vault will cap the TTL at a value
+that will not cause the token to exist beyond the max lease TTL.
```

Comment:
From a few versions now, the term `lease_duration` is replaced by `ttl` to avoid misinterpreting that `lease_duration` only belongs to leases and not tokens. Many endpoints simply refer to these as `ttl` and `max_ttl`. So, referring the "max lease TTL" in this documentation segment as just "maximum TTL", even though the variable name is `max_lease_ttl` (for historic reasons) makes sense.

@jefferai Thoughts?